types/ui.ts

```diff
@@ -24,10 +22,6 @@ export type PanelTypeT =
   | "remind"
   | "delegation";
 
-export type VotePanelContentT = VoteT;
```

Comment:
when I initially added these types there were going to be more kinds of panel content. since there is only one now, these are redundant